### DISCUSSION TITLE
Trouble with the raspberry pi of elementary OS

### DISCUSSION THREAD

**Discussion by aXdey65i** [elementary/initial-setup#200] [Q&A] [answered]
Please help me!!

When I run the elementary OS on raspberry pi, a screen like this appears and I can't proceed!!

I'm in trouble. Please tell me how to solve it!!
![IMG_0094](https://github.com/user-attachments/assets/1c2ffa9d-9d64-4635-833d-67337f630909)

**Comment by ryonakano** [comment]
Does another device name work for you?

**Reply by aXdey65i** [reply]
Yes. This is a screen that has been turned on several times, and the device name and username are the same, but it remains on this screen all the time.
I saw your name, are you Japanese by any chance?
I'm sorry if it's different!!
By the way, I am Japanese.

**Reply by ryonakano** [reply, marked as answer]
> Yes. This is a screen that has been turned on several times, and the device name and username are the same, but it remains on this screen all the time.

I see. Thank you for your information.

I think your issue might be #182 because the fix against it is only released for OS 8.0 Daily and 8.0 RC, while the image for Raspberry Pi is built on OS 7.1 Daily (edit: opened https://github.com/elementary/os/issues/722). I guess rebasing the image on OS 8.0 RC would fix the issue, but I don't have Raspberry Pi so I can't test even if I fixed the issue.

> I saw your name, are you Japanese by any chance?

Yes, I live in Japan and speak Japanese as my tongue.

**Reply by aXdey65i** [reply]
成る程…raspberry piで8.1がリリースされるのを待った方がいいと言うことですね。ありがとうございます！
